Content/ReleaseBriefings/4_6/Section_GeneralEnhancements.tex

```diff
@@ -0,0 +1,154 @@
+%==========================================================================
+
+\begin{frame}[fragile]
+
+  {\Huge General Enhancements}
+
+  \vspace{10pt}
+
+\end{frame}
+
+%==========================================================================
+
+% Examples
+
+% note: always keep the [fragile] for your frames!
+
+%\begin{frame}[fragile]{Example list}
+%  \begin{itemize}
+%      \item Item 1
+%      \item Item 2 with some \texttt{code}
+%      \begin{itemize}
+%        \item Sub-item 2.1
+%        \item Sub-item 2.2
+%      \end{itemize}
+%  \end{itemize}
+%\end{frame}
+
+%\begin{frame}[fragile]{Example code}
+%    \begin{code}[keywords={std}]
+%        #include <iostream>
+%        
+%        int main() {
+%            std::cout << "hello world\n";
+%        }
+%    \end{code}
+%\end{frame}
+
+%\begin{frame}[fragile]{Example table}
+%    \begin{center}
+%        \begin{tabular}{l|l}
+%            a & b \\\hline
+%            c & d
+%        \end{tabular}
+%    \end{center}
+%\end{frame}
+
+%==========================================================================
+
+\begin{frame}[fragile]{Fix a warning from kokkos\_check}
+ \begin{itemize}
+    \item \texttt{kokkos\_check}: Check at configure time that Kokkos was built with the requested backends and target architectures. Throws a fatal error otherwise.
```

Comment:
```suggestion
    \item \texttt{kokkos\_check}: Check at configure time that Kokkos was built with the requested backends and target architectures.
```
Not necessarily true, we support passing `RETURN_VALUE TheResult`

Reply:
@tpadioleau